README.rst

```diff
@@ -62,6 +62,10 @@ Currently, Scatterkit supports the following analysis modules (alphabetically):
 
    * - Module
      - Description
+   * - DiporderStructureFactor
+     - Structure factor for dipoles
+   * - RDFDiporder
+     - Spherical radial distribution function between dipoles
```

Comment:
```suggestion
     - Radial distribution function between dipoles
```
I would ditch the qualifier spherical because we are not in MAICoS here. People will always assume the normal rdf and would be bewildered by us calling it spherical